judo-tatami-jsl-jsl2psm/src/test/resources/derived/TestDerivedExpressionModel.jsl

```diff
@@ -34,6 +34,11 @@ entity Lead {
     field Integer value = 100000;
     field Boolean closed;
     relation SalesPerson salesPerson opposite leads;
+    derived Integer orElseValue => self.value!orElse(value = 10)!abs()!abs();
```

Comment:
is `!abs()!abs()` intentional?

Reply:
yes, it is intentional. It tests the chained function calls with no side effect.